### DISCUSSION TITLE
Future of the Internet??

### DISCUSSION THREAD

**Discussion by Showmobud** [community/community#81939] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

If you was to be apart of a new social community what would be some of interest to you??

**Comment by Smbfuplayer** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> If you was to be apart of a new social community what would be some of interest to you??

- [x] - @Showmobud []()****

**Reply by Smbfuplayer** [reply]
> Touch and hold a clip to pin it. Unpinned clips will be deleted after 1 hour.~~~~

**Reply by Showmobud** [reply]
The functionality,the process,with intentions of making a self improvement aplacation that is personal,that pays you to reterive information, while staying connected, and become a smarter shopper

**Reply by karladointheemost** [reply]
I'm open to ideas and help

**Reply by karladointheemost** [reply]
It croleates a new type of  network style okeaencouraging teens to stay productive and smoke free.

**Comment by Showmobud** [comment]
I would like to share with any one who is interested in making the future of the Internet

**Reply by karladointheemost** [reply]
I'm in

**Reply by Showmobud** [reply]
How are you you?

**Reply by karladointheemost** [reply]
Ok and yourself

**Reply by nikolaikolev196909** [reply]
> I would like to share with any one who is interested in making the future of the Internet

**Comment by Real0xF** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> If you was to be apart of a new social community what would be some of interest to you??

Q: What should we do for future of the Internet

A: I'm looking for an opportunity to share true information for all, if i access all of the internet (If u like my suggest, maybe u will pin my suggest :))

**Reply by Mc160191** [reply]
- []()

**Reply by Showmobud** [reply]
if they are willing to work there are opportunities. With my new affiliate  centralized platform.internet money is unbelievable and even though it's personal and I want to earn I also want people to be knowledgeable. That the technology industry is willing to pay every one.that are willing to work and most likely just good people there aren't any more space for the bad 2024. I am a dreamer and innovator with a dream so big that it will impact everyone's life

**Comment by Showmobud** [comment]
I was wondering about smart contracts

**Reply by Showmobud** [reply]
I show here to day
